lib/gates_cirq.h

```diff
@@ -828,8 +924,11 @@ struct ISwapPowGate {
   }
 };
 
-// The (exponent=2*phi/pi, global_shift=0) instance of ISwapPowGate.
-// This is a function in Cirq.
+/**
+ * The `(exponent = 2*phi/pi, global_shift = 0)` instance of ISwapPowGate.
+ * This is equivalent to an ISWAP gate with a global phase.
```

Comment:
I think this is not equivalent to an ISWAP gate with a global phase. It is basically ISwapPowGate without a global shift.

Reply:
Done.